decidim-admin/app/helpers/decidim/admin/process_groups_for_select_helper.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module Decidim
+  module Admin
+    # This class contains helpers needed to format ParticipatoryProcessGroups
+    # in order to use them in select forms.
+    #
+    module ProcessGroupsForSelectHelper
+      # Public: A formatted collection of ParticipatoryProcessGroups to be used
+      # in forms.
+      #
+      # Returns an Array.
+      def process_groups_for_select
+        @process_groups_for_select ||=
+          [[I18n.t("decidim.participatory_processes.participatory_process_groups.none"), 0]] +
```

Comment:
I believe this can be done using `prompt` as option on the `select` helper, instead of adding it as a first option.